dotnet/src/webdriver/DriverService.cs

```diff
@@ -151,6 +154,12 @@ public int ProcessId
     /// </summary>
     public string? DriverServicePath { get; set; }
 
+    /// <summary>
+    /// Collects the driver log output and writes it to the console. Defaults to <see langword="true"/>.
+    /// Internal variable to avoid using the stream when logs are sent to a file.
+    /// </summary>
+    protected bool WriteDriverLogToConsole { get; set; } = true;
```

Comment:
I am pretty sure we don't need this property, despite its name is not clear. Just don't introduce it.

Reply:
Yeah, now with the approach it has now, this is unnecessary.